stubs/docopt/docopt.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Optional, Union
+
+__version__: str
+
+def docopt(
+    doc: str,
+    argv: Optional[list[str]] = ...,
+    help: Optional[bool] = ...,
+    version: Optional[Any] = ...,
+    options_first: Optional[bool] = ...,
+) -> dict[str, Union[str, bool]]: ...
```

Comment:
With a dict like this, you need to check whether `the_dict[some_key]` is `str` or `bool`, and that can be annoying. This is similar to what you find with ctrl+f "avoid Union return types" in CONTRIBUTING.md.

```suggestion
) -> dict[str, Any]: ...
```

Reply:
Right, that makes sense. Changed in the v3 I just force pushed to my fork.